src/Button/ToggleGroup/ToggleGroup.spec.jsx

```diff
@@ -103,9 +103,6 @@ describe('<ToggleGroup />', () => {
       allowDeselect: true
     });
 
-    wrapper.find(ToggleButton).first().simulate('click');
-    expect(wrapper.state().selectedName).toBe('Shinji');
```

Comment:
Is this not valid anymore?

Reply:
After changing the `allowDeselect` property it should return `null` since the button is already clicked